File: sdks/go/pkg/models/shared/foldermini.go
package shared

type FolderMiniTypeEnum string

const (
	FolderMiniTypeEnumFolder FolderMiniTypeEnum = "folder"
)

// FolderMini
// The bare basic representation of a folder, the minimal
// amount of fields returned when using the `fields` query
// parameter.
type FolderMini struct {
	Etag       *string                `json:"etag,omitempty"`
	ID         string                 `json:"id"`
	Name       *string                `json:"name,omitempty"`
	SequenceID map[string]interface{} `json:"sequence_id,omitempty"`
	Type       FolderMiniTypeEnum     `json:"type"`
}

File: sdks/java/lib/src/main/java/models/shared/FolderMini.java
package .models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * FolderMini
 * The bare basic representation of a folder, the minimal
 * amount of fields returned when using the `fields` query
 * parameter.
**/
public class FolderMini {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("etag")
    public String etag;
    public FolderMini withEtag(String etag) {
        this.etag = etag;
        return this;
    }
    @JsonProperty("id")
    public String id;
    public FolderMini withId(String id) {
        this.id = id;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("name")
    public String name;
    public FolderMini withName(String name) {
        this.name = name;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("sequence_id")
    public java.util.Map<String, Object> sequenceId;
    public FolderMini withSequenceId(java.util.Map<String, Object> sequenceId) {
        this.sequenceId = sequenceId;
        return this;
    }
    @JsonProperty("type")
    public FolderMiniTypeEnum type;
    public FolderMini withType(FolderMiniTypeEnum type) {
        this.type = type;
        return this;
    }
}